### PR TITLE
Fix variable mistaches

### PR DESCRIPTION
**Pull request by crodas**
Attempt to fix fatal error when importing a multisig with a QR code.

image

### CODE REVIEW

File: ports/stm32/boards/Passport/modules/wallets/multisig_import.py [hunk 1]
```diff
@@ -16,10 +16,10 @@
 
 async def read_multisig_config_from_qr():
     from pages import ScanQRPage
-    result = await ScanQRPage(card_header={'title': 'Multisig Import'}).show()
+    data = await ScanQRPage(card_header={'title': 'Multisig Import'}).show()
 
-    if result is not None or result.error is not None:
-        selt.set_result(False)
+    if data is not None and data.error is not None:
+        data.set_result(False)
```
Comment (jeandudey) [inline ports/stm32/boards/Passport/modules/wallets/multisig_import.py:22]: Hello!

Thanks for taking the time to fix this up, just a nit:

```suggestion
        self.set_result(False)
```

**Comment by jeandudey** [comment]
Hello,

Thanks for taking the time to fix this up!

The fix for that error is coming on the 2.0.4 release.

The change resides in the `dev-2.0.4` [branch](https://github.com/Foundation-Devices/passport2/blob/dev-v2.0.4/ports/stm32/boards/Passport/modules/wallets/multisig_import.py#L17-L19).

**Comment by crodas** [comment]
Thanks for a prompt fix @jeandudey. Is there a release date for `2.0.4`?

**Comment by jeandudey** [comment]
@crodas it should be released in the next couple of days.

**Pull request closed without merging**